app/src/main/java/com/example/babytracking/HealthActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.babytracking
```

Comment:
As a follow up (don't do it as part of this PR or it will confuse Git), you should update the package name to something else. Typically, it's based on your domain name. Here, you could do something like `com.nicobir.babytracking`?